Dangerfile

```diff
@@ -0,0 +1,26 @@
+# Warn when there is a big PR
+warn("Big PR") if git.lines_of_code > 500
+
+# Sometimes its a README fix, or something like that - which isn't relevant for
+# including in a CHANGELOG for example
+has_app_changes = !git.modified_files.grep(/Source/).empty?
+has_test_changes = !git.modified_files.grep(/Tests/).empty?
+
+# Add a CHANGELOG entry for app changes
+if !git.modified_files.include?("CHANGELOG.md") && has_app_changes
+  fail("Please include a CHANGELOG entry to credit yourself! \nYou can find it at [CHANGELOG.md](https://github.com/realm/SwiftLint/blob/master/CHANGELOG.md).")
+    markdown <<-MARKDOWN
+Here's an example of your CHANGELOG entry:
+```markdown
+* #{github.pr_title}#{' '}
```

Comment:
I've kept the `#{' '}` to make sure we don't miss this if someone uses an editor that strips trailing whitespaces.